README-fr.md

```diff
@@ -157,4 +157,8 @@ Maintenez l'implication en étant conviviaux à distance:
 
 ![Bonnes pratiques en télétravail – #1 La gestion du temps | Infographie](https://media-exp1.licdn.com/dms/image/C4D22AQGObb6san-TJw/feedshare-shrink_2048_1536-alternative/0?e=1605744000&v=beta&t=UDMNzZnHGSo06aUOkvisbnasQjv7EffV2R5rg5m-Mc4)
 
+## Shirley
```

Comment:
hey 👋 

Perhaps we could add some information to engage the conversation with her. 
We could add a mention to her twitter account... something like `Shirley - @ShirleyAlmCh`
What do you think? 🤔